metis-normalization/src/test/java/eu/europeana/normalization/dates/extraction/dateextractors/NumericPartsDateExtractorTest.java

```diff
@@ -49,13 +51,15 @@ void extractDateSpaces(String input, String expected) {
   }
 
   void extract(String input, String expected, DateNormalizationExtractorMatchId dateNormalizationExtractorMatchId) {
```

Comment:
suggestion: rename to ` assertNumericPartsDateExtractor`

Reply:
Done. For consistency I've kept it similar to other tests.